### DISCUSSION TITLE
[help] Disable the spinner

### DISCUSSION THREAD

**Discussion by Kodiologist** [ropensci/targets#1051] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

How can I turn off the spinner (i.e., the little animation of the characters `-`, `\`, `|`, `/`) during `tar_make`? I think it's interfering with what some of my targets are trying to show on the console. The project I'm working with is currently on `targets` 0.14.1.

**Comment by wlandau** [comment, marked as answer]
You can turn off the spinner with `callr_arguments = list(spinner = FALSE)` in `tar_make()`.
